docs/rules/spin-button-unrecommended-labelling-v9.md

```diff
@@ -0,0 +1,68 @@
+# Accessibility: Unrecommended accessibility labelling - SpinButton (`@microsoft/fluentui-jsx-a11y/spin-button-unrecommended-labelling-v9`)
+
+<!-- end auto-generated rule header -->
+
+All interactive elements must have an accessible name.
+
+Spin Button components need a visual label.
+
+Using aria-label or wrapping the SpinButton in a Tooltip component is not recommended.
+
+<https://www.w3.org/TR/html-aria/>
+
+
+## Rule Details
+
+This rule aims to...
+
+Examples of **unrecommended** code for this rule:
+
+```jsx
+<SpinButton defaultValue={10} min={0} max={20} aria-label={"my-aria-label-1"} />
+```
+
+```jsx
+
+<Tooltip content="Sping Button label" relationship="label">
```

Comment:
Can we remove this from the readme as tooltip is not recommended for spin button, we should not recommend this through the readme in my pov, @aubreyquinn what are your thoughts on this?

we should have the rule, but lets avoid it in readme.

Reply:
This talks about unrecommended ways of doing this. Same as other readmes talk about incorrect ones

Reply:
Yes, I agree :)